README.md

```diff
@@ -29,92 +31,92 @@ materials science, quantum chemistry, and biology.
 * [Citing DeepChem](#citing-deepchem)
 
 ## Requirements
-* [pandas](http://pandas.pydata.org/)
+
+DeepChem requires these packages on any condition.
+
 * [joblib](https://pypi.python.org/pypi/joblib)
-* [sklearn](https://github.com/scikit-learn/scikit-learn.git)
-* [numpy](https://store.continuum.io/cshop/anaconda/)
+* [pandas](http://pandas.pydata.org/)
+* [Pillow](https://pypi.org/project/Pillow/)

```

Comment:
I seem pillow is a hard requirement. 
https://github.com/deepchem/deepchem/blob/7e0f1771b5bd0c1894da6bc5cd1f84639ccda1ef/deepchem/data/data_loader.py#L21
https://github.com/deepchem/deepchem/blob/3467db1e9d7a35e624c04f90d6bfc89436f246d6/deepchem/trans/transformers.py#L14